pico/server.py

```diff
@@ -6,10 +6,13 @@
 from werkzeug.middleware.shared_data import SharedDataMiddleware
 from werkzeug.utils import import_string
 
+DEF_SERVER_IP   = '127.0.0.1'
```

Comment:
+1 for using constants for these but please let's use `DEFAULT_SERVER_IP` & `DEFAULT_SERVER_PORT`. Or maybe simply `DEFAULT_IP` & `DEFAULT_PORT`?

Reply:
Hi Fergal, Added a new commit with the changes you requested.